### PR TITLE
Improve xcm integration test with CI

### PR DESCRIPTION
**Pull request by yrong**
Require https://github.com/Snowfork/polkadot-sdk/pull/20

### CODE REVIEW

File: parachain/pallets/inbound-queue/src/lib.rs [hunk 1]
```diff
@@ -130,8 +130,8 @@ pub mod pallet {
 	#[pallet::event]
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T> {
-		/// A message was received from Ethereum
-		MessageReceived {
+		/// Xcm message sent to dest chain
+		MessageSent {
```
Comment (vgeddes) [inline parachain/pallets/inbound-queue/src/lib.rs:134]: This doesn't need to be renamed. The inbound queue really did receive a message.

The XCM subsystem should already emit an `XcmSent` event or something like that.

Comment (yrong) [reply parachain/pallets/inbound-queue/src/lib.rs:134]: Or can we rename as `MessageProcessed`?

Comment (vgeddes) [reply parachain/pallets/inbound-queue/src/lib.rs:134]: Look at it this way, the bridge has several layered protocols:

1. The lowest layer is the point-to-point link between the Gateway and in the inbound queue pallet, powered by light clients.
2. The layer above is XCM, which provides a link from the Gateway to any parachain on Polkadot
3. Finally, there is a the application logic built on top of XCM, be it token transfers, or Transact

Logically, this `MessageReceived` is the correct naming for layer (1). A messages was sent by the gateway, and received by the inbound queue pallet.

To track the execution of messages in layer 2, we should use whatever events are emitted by the various XCM pallets, along with the Xcm Hash Ids.

**Review by alistair-singh — APPROVED** [review]
Nice work!

File: parachain/pallets/inbound-queue/src/lib.rs
Comment (vgeddes) [inline parachain/pallets/inbound-queue/src/lib.rs:1]: I'm currently touching this part of the code, and these non-essential refactorings will get in the way. Can we reduce the scope of this PR to just adding the integration tests?

**Review by vgeddes — APPROVED** [review]
Looks great!

**Pull request merged**